ingestion/src/metadata/utils/helpers.py

```diff
@@ -29,6 +30,10 @@
 logger = utils_logger()
 
 
+def random_int(min_val=0, max_val=999):
```

Comment:
I am not sure it is a good idea to have default value. Picking 0 and 999 is arbitrary to me and it makes code less readable (as there is no real logic).

Reply:
reverted the above change, thanks